mission.sqm

```diff
@@ -32309,16 +32310,341 @@ class Mission
 			id=1098;
 			type="C_SUV_01_F";
 		};
+		class Item365
+		{
+			dataType="Logic";
+			class PositionInfo
+			{
+				position[]={473.59708,0,776.08887};
+			};
+			name="F3_preMount_FIA";
+			init="[synchronizedObjects this,true,false] call f_fnc_mountGroups;";
```

Comment:
Is the order of `synchronizedObjects` stable?
If so, is it easy for a mission maker to determine and change the order?

Reply:
In my testing the order was always deterministic.
You can't really see the order in the editor, but recreating the synchronizations will change the order.
The first synchronized object is also the first group that gets moved into the vehicle.

Reply:
So if a mission maker wants ASL to always be the driver of a vehicle (assuming that slot was slotted) they have to ensure that group is sync first?

Reply:
Yes, but I already created all synchronizations in the correct order, so that SLs will be driver.
This behaviour is also described on the wiki page:
`When returning the synchronized objects, they are returned in the order that they were placed on the map `
<https://community.bistudio.com/wiki/synchronizedObjects>